### ISSUE TITLE
new async exception

### ISSUE THREAD

**Issue by darcosion** [soxoj/maigret#90]
Hello o/

I have now this new exception with a fresh install of maigret : 
```
[-] Starting a search on top 500 sites from the Maigret database...
[!] You can run search by full list of sites with flag `-a`
[*] Checking username someuser on:
Traceback (most recent call last):
  File "/usr/local/bin/maigret", line 11, in <module>
    load_entry_point('maigret==0.1.15', 'console_scripts', 'maigret')()
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.15-py3.6.egg/maigret/maigret.py", line 423, in run
    loop.run_until_complete(main())
  File "/usr/lib/python3.6/asyncio/base_events.py", line 484, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.15-py3.6.egg/maigret/maigret.py", line 355, in main
    max_connections=args.connections,
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.15-py3.6.egg/maigret/checking.py", line 559, in maigret
    results = await executor.run(coroutines)
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.15-py3.6.egg/maigret/checking.py", line 54, in run
    results = await self._run(tasks)
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.15-py3.6.egg/maigret/checking.py", line 123, in _run
    for _ in range(self.workers_count)]
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.15-py3.6.egg/maigret/checking.py", line 123, in <listcomp>
    for _ in range(self.workers_count)]
AttributeError: module 'asyncio' has no attribute 'create_task'
[base_events.py:1285] ERROR  13:13:40 Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f2f9e880f28>
sys:1: RuntimeWarning: coroutine 'ClientSession._request' was never awaited
```

**Comment by soxoj** [comment]
Well, looks like a side effect of the recent async scan improvement.

You are still using Python 3.6.9, am I right? I highly recommend you update to python 3.7.

I'll fix the problem and create a test, but I think that there will be other similar problems with python 3.6 support in the future.

**Comment by soxoj** [comment]
Fixed in #91, please check.

**Comment by darcosion** [comment]
It's work, thanks :+1:
